backend/accounts/account.go

```diff
@@ -33,15 +33,33 @@ type AddressList struct {
 	Addresses  []Address
 }
 
+// TextMemo represents a slip-0024 text memo.
+type TextMemo struct {
+	Note string `json:"text"`
+}
+
+// PaymentRequest contains the data needed to fulfill a slip-0024 payment request.
+// Text memos are the only memo type supported, currently.
+type PaymentRequest struct {
```

Comment:
why does this have json tags? It's not json serialized/deserialized is it?

Reply:
Leftover from the initial implementation :smile: